Documentation/ceph-upgrade.md

```diff
@@ -379,28 +420,24 @@ when your application pods will be affected by the CSI driver restart.
 For each node:
 - Drain your application pods from the node
 - Delete the CSI driver pods on the node
-  - The pods to delete will be named with a `csi-cephfsplugin` or `csi-rbdplugin` prefix and have a random suffix on each node.
-    However, no need to delete the provisioner pods: `csi-cephfsplugin-provisioner-*` or `csi-rbdplugin-provisioner-*`
+  - The pods to delete will be named with a `csi-cephfsplugin` or `csi-rbdplugin` prefix and have a
+    random suffix on each node. However, there is no need to delete the provisioner pods:
+    `csi-cephfsplugin-provisioner-*` or `csi-rbdplugin-provisioner-*`
   - The pod deletion causes the pods to be restarted and updated automatically on the node
+- Undrain application pods from the node
 
+## 7. Move Rook-Ceph Operator configs from Deployment Envs to the new ConfigMap (optional)
```

Comment:
Not all settings are supported in the configmap yet so it's too early to recommend moving all the settings. Also, since this is completely optional and the env vars are intended to be supported in the long term, I don't think it should be recommended as part of the upgrade guide. If they want to try the new feature they can look at the release notes and docs for new features.

Reply:
I removed this then.